core/controllers/profile.py

```diff
@@ -311,6 +347,36 @@ def get(self):
 class SignupHandler(base.BaseHandler):
     """Provides data for the editor prerequisites page."""
 
+
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'POST': {
+            'username': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            },
+            'agreed_to_terms': {
+                'schema': {
+                    'type': 'bool'
+                },
+            },
+            'default_dashboard': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            },
```

Comment:
For this I think there are two options, so probably specify those.

Reply:
Done, Please have a look.

Reply:
Not done.

Reply:
I have tried to add those two options . Please tell me if I am missing anything.